modelina-website/src/components/playground/options/JavaGeneratorOptions.tsx

```diff
@@ -159,8 +192,15 @@ class JavaGeneratorOptions extends React.Component<
             />
           </label>
         </li>
-        <li>
-          <label className="flex items-center py-2 justify-between cursor-pointer">
+        <li className=' flex items-center'>
+          <InfoModal text="Include Overwrite HashCode Support :">
+            <p>
+            In Java, the hashCode() method is used to generate a unique numeric value (hash code) for an object. The default implementation of hashCode() in the Object class generates hash codes based on the memory address of the object, which may not be suitable for all classes.
+            <br/><br/>
+            By enabling the option to overwrite hashCode support, you can provide your own implementation of the hashCode() method in a class. This allows you to define a hash code generation algorithm based on the specific properties and requirements of the class.
```

Comment:
This is very generic wording about what toString is, however it does not explain what it does to the code generation.

Reply:
fixed!! :tada: